tests/pass-dep/shims/libc-fs.rs

```diff
@@ -135,3 +145,19 @@ fn test_readlink() {
     assert_eq!(res, -1);
     assert_eq!(Error::last_os_error().kind(), ErrorKind::NotFound);
 }
+
+fn test_o_tmpfile_flag() {
+    let dir_path = prepare_dir("miri_test_fs_dir");
+    create_dir(&dir_path).unwrap();
+    // test that the `O_TMPFILE` custom flag gracefully errors instead of panicking
```

Comment:
```suggestion
    // test that the `O_TMPFILE` custom flag gracefully errors instead of stopping execution
```
it wasn't panicking before, it was halting the interpreter